src/spec-configuration/containerFeaturesConfiguration.ts

```diff
@@ -650,6 +681,17 @@ async function fetchFeatures(params: { extensionPath: string; cwd: string; outpu
 			const featureDebugId = `${feature.consecutiveId}_${sourceInfoType}`;
 			params.output.write(`* Fetching feature: ${featureDebugId}`);
 
+			if (sourceInfoType === 'oci') {
+				params.output.write(`Fetching from OCI`, LogLevel.Trace);
+				await mkdirpLocal(featCachePath);
+				const success = await fetchOCIFeature(params.output, params.env, featureSet, ociCacheDir, featCachePath);
+				if (!success) {
+					// TODO: FAIL
```

Comment:
If we don't know how exactly to fail yet, at the very least should add a `LogLevel.Error` so we can catch it later?

Reply:
Added.